integration_tests/bindpy_04.py

```diff
@@ -102,8 +102,8 @@ def test_2D_array_bools():
 # Complex
 def test_array_complexes():
     n: Const[i32] = 5
-    a: c64[n] = empty([n], dtype=complex64)
-    b: c64[n] = empty([n], dtype=complex64)
+    a: c64[n] = empty([n], dtype=complex128)
+    b: c64[n] = empty([n], dtype=complex128)
```

Comment:
Good catch. It's confusing, but I think it's less confusing to use c32 and c64 to mean 32bit and 64bit complex numbers (each component), so c32 corresponds to f32 and c64 to f64. NumPy uses the size of the whole complex number (both components together). Now LPython catches the mistake, so that's good.